index.html

```diff
@@ -105,128 +105,145 @@ <h3 class="functions-text__title">Wesprzyj nas</h3>
+                            </div>
                         </div>
-                        <div class="about-us-social-box">
-                            <ul class="social-icons">
-                                <li>
-                                    <a href="#">
-                                        <span class="fa fa-facebook-f"></span>
-                                    </a>
-                                </li>
-                                <li>
-                                    <a href="#">
-                                        <span class="fa fa-twitter"></span>
-                                    </a>
-                                </li>
-                                <li>
-                                    <a href="#">
-                                        <span class="fa fa-github"></span>
-                                    </a>
-                                </li>
-                            </ul>
 
+                        <div class="about-us-inner-box">
+                            <div class="about-us-image">
+                                <img src="images/Kamil.jpg" alt="zdjęcie Kamila">
+                            </div>
+                            <div class="about-us-text">
+                                <div class="about-us-description">Slusznie nagrodzony tytułem "Najlepszy Prezes Fundacji
+                                    2018." Dla Kamila nie ma rzeczy niemożliwych. Prywatnie posiada 5 psów i 13 kotów.
+                                </div>
+                                <div class="about-us-social-box">
+                                    <ul class="social-icons">
+                                        <li>
+                                            <a href="https://www.facebook.com/" target="_blank">
+                                                <span class="fa fa-facebook-f"></span>
+                                            </a>
+                                        </li>
+                                        <li>
+                                            <a href="https://twitter.com/" target="_blank">
+                                                <span class="fa fa-twitter"></span>
+                                            </a>
+                                        </li>
+                                        <li>
+                                            <a href="https://github.com/" target="_blank">
+                                                <span class="fa fa-github"></span>
+                                            </a>
+                                        </li>
+                                    </ul>
+                                </div>
+                            </div>
+                            <div class="about-us-name">
+                                <h3>Kamil Lubowicki</h3>
+                                <p>Prezes Zarządu</p>
+                            </div>
```

Comment:
chyba nie trzymasz się tu w ogóle BEM'u który macie w DoD. Fajnie byłoby to zmienić. Na kanał merytoryka na slacku wrzuciłem artykuł który jest uzupełnieniem tego co mieliście na zajęciach z Anią. Może Ci to pomoże a jak nie to daj znać, spróbuje Ci jakoś pomóc

Reply:
Hej Paweł, zrobiłam ostatnie zmiany  zanim wrzuciłeś dodtakowe informacje o BEM-ie. Postaram się pozmieniać według tej metody.